contracts/base/LoansBase.sol

```diff
@@ -420,6 +430,11 @@ contract LoansBase is LoansInterface, Base {
         if (totalOwed == 0) {
             loans[loanID].status = TellerCommon.LoanStatus.Closed;
 
+            bytes32 userId = loans[loanID].userId;
+            if (userId != bytes32(0)) {
+                userIdHasLoan[userId] = false;
+            }
```

Comment:
We'll need to put this when a loan is liquidated as well since the loan is also repaid their.